string filename
bool record
float32 timeout_in_sec
---
bool success
